tests/virtctl/imageupload.go

```diff
@@ -34,7 +34,7 @@ const (
 	pvcSize = "100Mi"
 )
 
-var _ = Describe("[sig-storage][virtctl]ImageUpload", decorators.SigStorage, func() {
+var _ = Describe("[sig-storage][virtctl]ImageUpload", decorators.SigStorage, Serial, func() {
```

Comment:
```suggestion
var _ = Describe("[sig-storage][Serial][virtctl]ImageUpload", decorators.SigStorage, Serial, func() {
```

This have also existed here originally: [`36be728` (#12925)](https://github.com/kubevirt/kubevirt/pull/12925/commits/36be728df5618e16bfd78a5ea244d3e46623de05#diff-167e9ebe2dd43d34a0f106feeae260a41a00d7cecdc9dc8cc4cd5aea654c08a1L49)

Reply:
Actually that's on purpose. We're pretty [unhappy](https://github.com/kubevirt/kubevirt/pull/13194#issuecomment-2459496047) duplicating the serial decorator. In fact, so unhappy that I've contributed the change to ginkgo - https://github.com/onsi/ginkgo/pull/1484

Reply:
Awesome :)
Waiting for your PR to consume the new release and remove all `[Serial]`s :wink: